build_deploy.rhel.sh

```diff
@@ -16,7 +16,7 @@ docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
 docker build  --no-cache \
```

Comment:
This file is not required.

Reply:
It is been referred from app-interface, I will remove once that has been updated to refer build_deploy.sh.

Reply:
Ok, please make sure that it is cleaned up